### PR TITLE
i18n sync down out 04/24

### PR DESCRIPTION
**Pull request by Hamms**

### CODE REVIEW

File: apps/lib/blockly/tg_tj.js [hunk 1]
```diff
@@ -18,14 +18,14 @@ Blockly.Msg.COLOUR_PICKER_TOOLTIP = "Choose a color from the palette.";
...
+Blockly.Msg.COLOUR_RGB_GREEN = "green";
 Blockly.Msg.COLOUR_RGB_HELPURL = "http://www.december.com/html/spec/colorper.html";
-Blockly.Msg.COLOUR_RGB_RED = "сурх";
+Blockly.Msg.COLOUR_RGB_RED = "red";
 Blockly.Msg.COLOUR_RGB_TITLE = "color with";
 Blockly.Msg.COLOUR_RGB_TOOLTIP = "Create a color with the specified amount of red, green, and blue.  All values must be between 0 and 100.";
-Blockly.Msg.CONFIRM_CREATE_VARIABLE = "Сохтан";
-Blockly.Msg.CONFIRM_DELETE_FUNCTION_MESSAGE = "Огоҳӣ! Дар ҳақиқат мехоҳед %1 -ро кӯр кунед?";
+Blockly.Msg.CONFIRM_CREATE_VARIABLE = "Create";
+Blockly.Msg.CONFIRM_DELETE_FUNCTION_MESSAGE = "Warning! Are you sure you want to delete %1?";
```
Comment (Hamms) [inline apps/lib/blockly/tg_tj.js:28]: These changes all appear to be reverts; I'm unsure why.

File: apps/lib/blockly/es_mx.js [hunk 2]
```diff
@@ -200,7 +200,7 @@ Blockly.Msg.LOGIC_TERNARY_IF_FALSE = "si es falso";
 Blockly.Msg.LOGIC_TERNARY_IF_TRUE = "si es verdadero";
 Blockly.Msg.LOGIC_TERNARY_TOOLTIP = "Comprobar la condición en \"probar\". Si la condición es verdadera, devolver el valor \"si es cierto\"; de lo contrario, devolver el valor \"si es falso\".";
 Blockly.Msg.MATH_ADDITION_SYMBOL = "+";
-Blockly.Msg.MATH_ARITHMETIC_HELPURL = "http://es.wikipedia.org/wiki/Aritmética";
+Blockly.Msg.MATH_ARITHMETIC_HELPURL = "http://en.wikipedia.org/wiki/Arithmetic";
```
Comment (tanyaparker) [inline apps/lib/blockly/es_mx.js:203]: This one is right. The Spanish translation was submitted two years ago and the English translation (the correct one) a year ago. What's interesting is why it updated only now...

https://crowdin.com/translate/codeorg/54/enus-esmx#3352

File: apps/lib/blockly/tg_tj.js [hunk 3]
```diff
@@ -1,14 +1,14 @@
 Blockly.Msg.ACTUAL = "call";
-Blockly.Msg.ADD = "Илова";
-Blockly.Msg.ADD_COMMENT = "Иловаи шарҳ";
-Blockly.Msg.ADD_PARAMETER = "Иловаи параметр";
+Blockly.Msg.ADD = "Add";
+Blockly.Msg.ADD_COMMENT = "Add Comment";
```
Comment (tanyaparker) [inline apps/lib/blockly/tg_tj.js:3]: This file reverting to English is wrong.

https://crowdin.com/translate/codeorg/54/enus-tg#3187

File: apps/lib/blockly/tg_tj.js (hunk 1 above)
Comment (tanyaparker) [reply apps/lib/blockly/tg_tj.js:28]: Yeah... very strange. I wonder if you download the file straight from Crowdin, you should be able to confirm that it downloads with the translations. In which case it's probably something with our out script and where we merge/use fallbacks?

**Comment by Hamms** [comment]
Closing in favor of https://github.com/code-dot-org/code-dot-org/pull/14654

**Pull request closed without merging**